config/config.exs

```diff
@@ -18,6 +18,15 @@ config :g_searcher, GSearcherWeb.Endpoint,
   pubsub_server: GSearcher.PubSub,
   live_view: [signing_salt: "WiEvKDKO"]
 
+config :ueberauth, Ueberauth,
+  providers: [
+    google: {Ueberauth.Strategy.Google, []}
+  ]
+
+config :ueberauth, Ueberauth.Strategy.Google.OAuth,
+  client_id: System.get_env("GOOGLE_CLIENT_ID"),
+  client_secret: System.get_env("GOOGLE_CLIENT_SECRET")
```

Comment:
Let move the `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` to the `prod.secret.ex` file instead of keep it in the config.exs file, 2 files are different purpose @bterone ;)  

https://hexdocs.pm/phoenix/deployment.html#handling-of-your-application-secrets

Reply:
Thanks! This is useful to know. We don't have this on GCP 💨